src/network/tarcap/packets_handlers/common/get_blocks_request_type.hpp

```diff
@@ -0,0 +1,7 @@
+#pragma once
```

Comment:
do we really need separate file for this ?

Reply:
yes we do otherwise there was a circular dependency with includes